api/clients/v2/metrics/accountant.go

```diff
@@ -57,6 +67,10 @@ func (m *AccountantMetrics) RecordCumulativePayment(accountID string, wei *big.I
 	m.CumulativePayment.WithLabelValues(accountID).Set(gweiFloat64)
 }
 
+func (m *AccountantMetrics) RecordReservationPayment(accountID string, remainingCapacity float64) {
+	m.ReservationRemainingCapacity.WithLabelValues(accountID).Set(remainingCapacity)
```

Comment:
maybe add a note that we are only expecting max 100s of reservation, so the amount of accountID (prom dimensions) won't be excessively large

Reply:
9f13783a458a5a9855cec2516f29ea2c32803c7a

Went ahead and changed to a gauge